test/Numerics/SystemSolvers/columnwiselu.jl

```diff
@@ -56,7 +56,7 @@ let
     d_F = ArrayType(AB)
     d_F = DGColumnBandedMatrix{
         3,
-        N,
+        (N, N, N),
```

Comment:
Does the columnwise solver work with variable `N` yet?

Reply:
Yup, the variable degree acoustic wave test uses 1d imex with the LU solver

Reply:
Should we unit test it though?

Reply:
I may need some help coming up with a good unit test, but I can try to see if I can come up with something